packages/scripts/vue.config.js

```diff
@@ -0,0 +1,40 @@
+// vue.config.js
+module.exports = {
+  filenameHashing: false,
+  outputDir: 'dist',
+  configureWebpack: {
+    devtool: 'source-map',
+    externals: {
+      'bootstrap': 'bootstrap'
+    },
+    output: {
+      filename: 'js/scripts/[name].js',
+      chunkFilename: 'js/scripts/[name].js'
+    }
+  },
+  css: {
+    extract: {
+      filename: 'css/scripts/[name].css',
+      chunkFilename: 'css/scripts/[name].css'
+    }
+  },
+  'devServer': {
+    'proxy': {
+      '^/api': {
```

Comment:
i think you can combine these

Reply:
I tried a few things. But the redirect to the job handler keeps breaking every time i change this.